feat(lsp): added dev-tools.nvim (#1509)
AstroNvim/astrocommunity: lua/astrocommunity/lsp/dev-tools-nvim/README.md

# Dev-tools.nvim

> In-process LSP server of custom code actions, enhanced actions picker, community actions library and a convenient interface for customization and enhancement of your Neovim with custom code actions.

Reviewer: ```suggestion
Friendly developer tools for Nvim - Code Actions LSP server and library
```
